collector/receiver/otelarrowreceiver/internal/arrow/arrow.go

```diff
@@ -71,6 +73,8 @@ type Receiver struct {
 	authServer  auth.Server
 	newConsumer func() arrowRecord.ConsumerAPI
 	netReporter netstats.Interface
+	recvInFlightBytes metric.Int64UpDownCounter
+	recvInFlightItems metric.Int64UpDownCounter
```

Comment:
Would like to suggest, while you're here, that we count the number of in-flight requests, so an updowncounter with +1 / -1 increments. (Unless, that is, you're aware of some other instrumentation we have that does this!)

Reply:
Added a metric to track in flight requests